### ISSUE TITLE
CAO positions are not eventually consistent

### ISSUE THREAD

**Issue by anon55555** [minetest/minetest#11508]
CAO positions sometimes never update after a object moves, I think this is because the final position is sent unreliably.

A potential solution to this problem is to send a reliable position after an object stops moving (an object could be considered to stop moving when its position is the same on 2 consecutive steps, but differs from its position on the step before those 2 step).

**Comment by anon55555** [comment]
> A potential solution to this problem is to send a reliable position after an object stops moving (an object could be considered to stop moving when its position is the same on 2 consecutive steps, but differs from its position on the step before those 2 step).

This could still result in an incorrect CAO position because an unreliable packet could be received after the reliable one because unrels can arrive out-of-order.

**Comment by sfan5** [comment]
That is true but there are many such issues in Minetest that would need to be fixed by establishing a shared timeline between client and server one day.
